.github/workflows/code-style.yml

```diff
@@ -21,22 +21,33 @@ jobs:
 
       - name: Check Out Code
         uses: actions/checkout@v4
-        with:
-          fetch-depth: 1
 
-      - name: Get Composer Cache Directory
-        id: composer-cache
-        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+      - name: Download dependencies
+        uses: ramsey/composer-install@v3
+
+      - name: Run Tests
+        run: vendor/bin/psalm --output-format=github
```

Comment:
With `--output-format=github` it will make github review comments on the issues. 

(This may be detected automatically, but I could not find a reference for it in the docs)